pyproject.toml

```diff
@@ -13,7 +13,7 @@ langchain-server = "langchain.server:main"
 [tool.poetry.dependencies]
 python = ">=3.8.1,<4.0"
 pydantic = "^1"
-SQLAlchemy = "^1"
+SQLAlchemy = ">1.3,<3"
```

Comment:
not strictly related to this pr but should this be optional dependency? cc @hwchase17